SamplesDashboard/SamplesDashboard/Services/RepositoriesService.cs

```diff
@@ -140,22 +140,31 @@ public async Task<List<Node>> GetRepositories(string name, string endCursor = nu
             if (hasNextPage == true)
             {
                 var nextRepos = await GetRepositories(name, endCursor);
-                repos.AddRange(nextRepos);
-            }       
+                repositories.AddRange(nextRepos);
+            }
 
-            return repos;            
+            //remove localized repos to reduce repetition of samples
+            foreach (var repo in repositories.ToList())
```

Comment:
I think the better way of filtering out the localized repos would be use Regex to filter out names ending with the pattern `.xx-XX`.
This way we are sure the repo is a localization.